_pages/law-and-regs/design-standards/2010-stds.md

```diff
@@ -29,7 +29,7 @@ The Department has assembled this online version of the official 2010 Standards
 
 [2010 Standards for Public Accommodations and Commercial Facilities Title III]({{'/law-and-regs/design-standards/2010-stds/#2010-standards-for-public-accommodations-and-commercial-facilities-title-iii' | relative_url}})
 
-The Department has assembled into a separate publication the revised regulation guidance that applies to the Standards.  The Department included guidance in its revised ADA regulations published on September 15, 2010.  This guidance provides detailed information about the Department's adoption of the 2010 Standards including changes to the Standards, the reasoning behind those changes, and responses to public comments received on these topics.  The document, Guidance on the 2010 ADA Standards for Accessible Design, can be downloaded from [https://beta.ada.gov/law-and-regs/design-standards/standards-guidance/]( {{'/law-and-regs/design-standards/standards-guidance' | relative_url}})
+The Department has assembled into a separate publication the revised regulation guidance that applies to the Standards.  The Department included guidance in its revised ADA regulations published on September 15, 2010.  This guidance provides detailed information about the Department's adoption of the 2010 Standards including changes to the Standards, the reasoning behind those changes, and responses to public comments received on these topics.  The document, Guidance on the 2010 ADA Standards for Accessible Design, can be downloaded from [https://www/ada.gov/law-and-regs/design-standards/standards-guidance/]( {{'/law-and-regs/design-standards/standards-guidance' | relative_url}})

```

Comment:
Looks like the URL for ada.gov will display incorrectly. The '/' needs to be replaced by a '.'